recipes/winpcap/all/conanfile.py

```diff
@@ -0,0 +1,29 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class WinpcapConan(ConanFile):
+    name = "winpcap"
+    version = "4.1.2"
+    settings = "os", "compiler", "build_type", "arch"
+    description = "The WinPcap packet capture library."
+    homepage = "https://www.winpcap.org/"
+    url = "https://github.com/bincrafters/community"
+    license = "Muliple"
+
+    def configure(self):
+        if self.settings.os != "Windows":
+            raise ConanInvalidConfiguration("WinPcap is only supported for Windows. For other operating systems, look for libpcap.")
+
+    def package(self):
+        sha256 = "ea799cf2f26e4afb1892938070fd2b1ca37ce5cf75fec4349247df12b784edbd"
```

Comment:
This should be moved to a `conandata.yml` file. See https://github.com/bincrafters/templates/blob/1ab1ad862f042de9fd806f97e69fa74af4e2a5c6/recipes/default/all/conandata.yml#L1 and https://github.com/bincrafters/templates/blob/1ab1ad862f042de9fd806f97e69fa74af4e2a5c6/recipes/default/all/conanfile.py#L36 for examples how to use `conandata.yml` to download files

Reply:
I moved it to `conandata.yml` and also moved `tools.get()` to `def source(self)`